pkg/controller/import-controller.go

```diff
@@ -67,6 +70,30 @@ func (ic *ImportController) findImportPodFromCache(pvc *v1.PersistentVolumeClaim
 	return podList[0], nil
 }
 
+func (ic *ImportController) getPodEnvVar(pvc *v1.PersistentVolumeClaim) (importPodEnvVar, error) {
```

Comment:
Not a big deal, but I think it is generally preferable to return pointers when there may be an error, so `(*importPodEnvVar, error)` with first value being nil in error case.

Reply:
good idea. Thanks.